website/pages/docs/plugins/sources/awspricing/overview.mdx

```diff
@@ -0,0 +1,41 @@
+---
+name: AWS Pricing
+---
+# AWS Pricing Source Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+
+<Badge text={"Latest: " + getLatestVersion("source", `awspricing`)}/>
+
+The AWS Pricing Source plugin for CloudQuery extracts data from the [AWS Bulk Pricing Service](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/using-ppslong.html) and loads it into any supported CloudQuery destination (e.g. PostgreSQL, BigQuery, Snowflake, and [more](/docs/plugins/destinations/overview)).
+
+## Configuration
+
+The following configuration syncs from AWS Pricing service to a Postgres destination. The (top level) source spec section is described in the [Source Spec Reference](https://www.cloudquery.io/docs/reference/source-spec). The config for the `postgresql` destination is not shown here. See our [Quickstart](/docs/quickstart) if you need help setting up the destination.
+
+```yaml copy
+kind: source
+spec:
+  name: "awspricing"
+  path: "cloudquery/awspricing"
+  version: "VERSION_SOURCE_AWSPRICING"
+  tables: ["*"]
+  destinations: 
+    - "postgresql"
+  spec:
+```
+
+
+## AWS Pricing Spec
+
+This is the (nested) spec used by the AWS Pricing source plugin.
+- `region_codes` ([]string) (default: empty. Will sync data for all regions)
```

Comment:
Why `region_codes` and simply not `regions`, which would be more in-line with the aws plugin?

Reply:
Here are some of the accepted values (it varies based on service):
```
af-south-1-los-1
af-south-1
ap-east-1
ap-northeast-1-tpe-1
ap-northeast-1-wl1-kix1
ap-northeast-1-wl1-nrt1
ap-northeast-1
ap-northeast-2-wl1-cjj1
ap-northeast-2-wl1-sel1
ap-northeast-2
ap-northeast-3
ap-south-1-ccu-1
ap-south-1-del-1
ap-south-1
ap-south-2
ap-southeast-1-bkk-1
ap-southeast-1
ap-southeast-2-per-1
ap-southeast-2
ap-southeast-3
ap-southeast-4
ca-central-1-wl1-yto1
ca-central-1
eu-central-1-ham-1
eu-central-1-waw-1
eu-central-1-wl1-ber1
eu-central-1-wl1-dtm1
eu-central-1-wl1-muc1
eu-central-1
eu-central-2
eu-north-1-cph-1
eu-north-1-hel-1
eu-north-1
eu-south-1
eu-south-2
eu-west-1
eu-west-2-wl1-lon1
eu-west-2-wl1-man1
eu-west-2
eu-west-3
me-central-1
me-south-1-mct-1
me-south-1
sa-east-1
us-east-1-atl-1
us-east-1-bos-1
us-east-1-bue-1
us-east-1-chi-1
us-east-1-dfw-1
us-east-1-iah-1
us-east-1-lim-1
us-east-1-mci-1
us-east-1-mia-1
us-east-1-msp-1
us-east-1-nyc-1
us-east-1-phl-1
us-east-1-qro-1
us-east-1-scl-1
us-east-1-wl1-atl1
us-east-1-wl1-bna1
us-east-1-wl1-chi1
us-east-1-wl1-clt1
us-east-1-wl1-dfw1
us-east-1-wl1-dtw1
us-east-1-wl1-iah1
us-east-1-wl1-mia1
us-east-1-wl1-msp1
us-east-1-wl1-nyc1
us-east-1-wl1-tpa1
us-east-1-wl1-was1
us-east-1-wl1
us-east-1
us-east-2
us-gov-east-1
us-gov-west-1
us-west-1
us-west-2-den-1
us-west-2-las-1
us-west-2-lax-1
us-west-2-pdx-1
us-west-2-phx-1
us-west-2-sea-1
us-west-2-wl1-den1
us-west-2-wl1-las1
us-west-2-wl1-lax1
us-west-2-wl1-phx1
us-west-2-wl1-sea1
us-west-2-wl1
us-west-2
```